### DISCUSSION TITLE
Will it support running as a service ?

### DISCUSSION THREAD

**Discussion by xxnuo** [dani-garcia/vaultwarden#2688] [Ideas]
Running lightly standalone
Just like this [https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#service](https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#service)

Preview:
![image](https://user-images.githubusercontent.com/54252779/184520983-36f42789-3d93-416b-8f79-f5f232c69319.png)

**Comment by BlackDex** [comment]
Have you checked the Vaultwarden wiki? https://github.com/dani-garcia/vaultwarden/wiki/Setup-as-a-systemd-service

**Reply by xxnuo** [reply]
on windows it doesn't work, i need to setup the service manually.
my question is does it can provide a service controled by built-in commands.

**Reply by BlackDex** [reply]
Indeed, it is just start and stop by starting the application and killing it.
No install, restart, refresh indeed.
